hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/DBScanner.java

```diff
@@ -61,8 +61,9 @@
   private String tableName;
 
   @CommandLine.Option(names = {"--with-keys"},
+      required = true,
       description = "List Key -> Value instead of just Value.",
-      defaultValue = "false",
+      defaultValue = "true",
```

Comment:
Thanks @sky76093016 for working on this.
Do we need to change the default value of this option `--with-keys` to true?

Reply:
Thanks @errose28 for answering my question. I agree with you.